pkg/models/ut.go

```diff
@@ -0,0 +1,43 @@
+package models
+
+type UnitTestResult struct {
+	Status   string `yaml:"status"`
+	Reason   string `yaml:"reason"`
+	ExitCode int    `yaml:"exit_code"`
+	Stderr   string `yaml:"stderr"`
+	Stdout   string `yaml:"stdout"`
+	Test     string `yaml:"test"`
+}
+
+type UnitTestsDetails struct {
+	Language                       string     `yaml:"language"`
+	ExistingTestsFunctionSignature string     `yaml:"existing_test_function_signature"`
```

Comment:
Very long name. Lets try to keep name single word or max 2 words if possible.

Reply:
refactored